bin/xip-pdns

```diff
@@ -75,13 +75,14 @@ qtype_is() {
   [ "$QTYPE" = "$1" ] || [ "$QTYPE" = "ANY" ]
 }
 
-qname_is_root_domain() {
-  [ "$QNAME" = "$ROOT_DOMAIN" ]
+qname_is_rootish() {
```

Comment:
Opening to better names than "rootish" just thought it was pretty funny and somewhat self explanatory